apps/freeablo/falevelgen/levelgen.cpp

```diff
@@ -456,30 +456,30 @@ namespace FALevelGen
         {
             for (int32_t y = 0; y < (int32_t)level.height(); y++)
             {
-                if (level[x][y] != wall || (!includeBorders && borders(x, y, blank, level)))
+                if (level[x][y] != (int)Basic::wall || (!includeBorders && borders(x, y, Basic::blank, level)))
```

Comment:
I don't love these casts, but it's probably the best option atm. We could at least have it be a cast to int32_t though instead of int.